TestResult.py

```diff
@@ -24,39 +26,50 @@ def __init__(self, description, name=None):
             self.name = inspect.stack()[1][3]
         self.timer = time.time()
 
+    def _current_time(self):
+        return str(datetime.datetime.now().time())
```

Comment:
I think microseconds is a little excessive (and on my platform the least significant digits are always 333 oddly!)
How about this?
```python
        return datetime.datetime.now().time().isoformat('milliseconds')
```

Reply:
For some reason that complained about isoformat not allowing arguments, which is odd given what the docs say. Used an alternative way around it.

Reply:
Doesn't work in python 2.7, but works for me with python 3.7 and just tried online at https://www.tutorialspoint.com/execute_python3_online.php which is 3.6

Reply:
Ok, well I'm running Python 3.5 here (default for Ubuntu Xenial) which caused an exception.